eng/Versions.props

```diff
@@ -20,23 +20,35 @@
     <UsingToolVSSDK>true</UsingToolVSSDK>
   </PropertyGroup>
   <!-- Production Dependencies -->
-  <PropertyGroup>
+  <!-- Condition consumption of maintenance-packages dependencies based on source build.
+       This is to prevent "package downgrade" errors coming from other packages that are
+       already consuming the newest version of these same dependencies. -->
+  <PropertyGroup Condition="'$(DotNetBuildSourceOnly)' == 'true'">
+    <!-- Use newest package versions. -->
+    <MicrosoftIORedistVersion>6.1.0</MicrosoftIORedistVersion>
```

Comment:
It looks like this is not actually referenced in the SB configuration.  See https://github.com/dotnet/msbuild/blob/main/src/Directory.BeforeCommon.targets#L113

Reply:
Ah good catch. I can revert this line change. Thanks!

Reply:
Fixed.